.cookiecutter/includes/requirements/prod.in

```diff
@@ -0,0 +1,9 @@
+newrelic
+
+hubspot-api-client
+importlib_resources
+jinja2
+psycopg2
+SQLAlchemy
+sqlparse
+tabulate
```

Comment:
This is the exact list of production requirements that's [currently in `prod.in`](https://github.com/hypothesis/report/blob/main/requirements/prod.in).